src/Apps/Order/Components/TransactionDetailsSummaryItem.tsx

```diff
@@ -220,8 +231,16 @@ export const TransactionDetailsSummaryItem: FC<TransactionDetailsSummaryItemProp
         data-test="buyerTotalDisplayAmount"
       />
       <Spacer y={2} />
+      {offer?.shippingTotalCents &&
+      order.requestedFulfillment?.__typename === "CommerceShipArta" &&
+      order.mode === "OFFER" &&
+      order.displayState !== "PROCESSING" ? (
+        <Text variant="sm" color="black60">
+          *Estimate Only. Price may vary once offer is finalized.
+        </Text>
+      ) : null}
       <Text variant="sm" color="black60">
-        *Additional duties and taxes{" "}
+        <Sup>†</Sup> Additional duties and taxes{" "}
```

Comment:
Same comment as above, these are shared across all order types so we probably dont want to show the cross unless there is a "shipping" footnote too